src/services/websocket_server/mod.rs

```diff
@@ -223,20 +230,22 @@ async fn resubscribe(
                 })
                 // Above we want to resubscribe as quickly as possible so use a high concurrency value
                 // But here we prefer stability and are OK with a lower value
-                .buffer_unordered(50)
+                .buffer_unordered(25)
```

Comment:
Is the size not too low?

Reply:
This is the amount of parallel subscribe/publish operations to do at once. It doesn't need to be higher